src/biophysics.jl

```diff
@@ -370,4 +370,123 @@ function evap(
   J_evap = uconvert(u"g/s",J_evap)
 
   (Q_evap = Q_evap, J_evap = J_evap, J_resp = J_resp, J_cut = J_cut, J_eyes = J_eyes)
+end
+
+function resp(
+    T_x = (23+273.15)K,
+    mass = 0.04kg,
+    Q_metab = 0.01241022W,
+    O2_ext_ref = 20,
+    pant = 1,
+    rq = 0.8,
+    T_air = (20+273.15)K,
+    rh = 50,
+    P_atmos = 101325Pa,
+    pctO2 = 20.95,
+    pctCO2 = 0.03,
+    pctN2 = 79.02)
+  # C     COMPUTES RESPIRATORY HEAT AND WATER LOSS VIA MASS FLOW THROUGH THE LUNGS,
+  # C     GIVEN GAS CONCENTRATIONS, PRESSURE, RESPIRATION RATE AND HUMIDITY
+  #
+  # C     NOTE THAT THERE IS NO RECOVERY OF HEAT OR MOISTURE ASSUMED IN THE NOSE
+
+  # C     DEFINING VARIABLES
+  # C     P_atmos = BAROMETRIC PRESSURE (PA)
+  # C     O2_ext_ref = EXTRACTION EFFICIENCY (PER CENT)
+  # C     GEVAP = GRAMS OF WATER EVAPORATED FROM RESPIRATORY TRACT/S
+  # C     Q_resp = HEAT LOSS DUE TO RESPIRATORY EVAPORATION (W)
+  # C     R = UNIVERSAL GAS CONSTANT (PA-M3/MOL-K) = (J/MOL-K)
+  # C     rh = RELATIVE HUMIDITY (PER CENT)
+  # C     rq = RESPIRATORY QUOTIENT (MOL CO2 / MOL O2)
+  # C     TC = ANIMAL CORE TEMPERATURE(C)
+
+  # C     ASSIGNING REFERENCE VALUES TO VARIABLES
+  # C     AIR FRACTIONS FROM SCHMIDT-NIELSEN, 2ND ED. ANIMAL PHYSIOLOGY CITING
+  # C     OTIS, 1964
+
+  fO2_ref = 0.2095
+  fN2_ref = 0.7902
+  fCO2_ref = 0.0003
+  fO2 = fO2_ref
+  fN2 = fN2_ref
+  fCO2 = fCO2_ref
+  #C     ALLOWING USER TO MODIFY GAS VALUES FOR BURROW, ETC. CONDITIONS
+  if fO2 != pctO2 / 100
+    fO2 = pctO2 / 100
+  else
+    fO2 = fO2_ref
+  end
+  if fN2 != pctN2 / 100
+    fN2 = pctN2 / 100
+  else
+    fN2 = fN2_ref
+  end
+  if fCO2 != pctCO2 / 100
+    fCO2 = pctCO2 / 100
+  else
+    fCO2 = fCO2_ref
+  end
+
+  #C     ERROR CHECKING for % of each air constituent summing to 1
+  totgas = fO2 + fN2 + fCO2
+  if totgas  >  1
+    fO2 = 1 - (fN2 + fCO2)
+  end
+  if totgas  <  1
+    fO2 = 1 - (fN2 + fCO2)
+  end
+
+  P_O2 = P_atmos * fO2
+  V_O2_STP = ((1 / 3.6E+06) * (Unitful.ustrip(Q_metab) / 0.0056)/1000)m^3
```

Comment:
Seems like the `E+06` and `/1000` are redundant with the unit information somehow